qiskit/visualization/pulse/qcstyle.py

```diff
@@ -145,9 +146,9 @@ def __init__(self, figsize: Tuple[float, float] = (7, 5),
             num_points: Number of points for interpolation.
             dpi: Resolution in the unit of dot per inch to save image.
         """
-        self.figsize = figsize
-        self.title_font_size = title_font_size
-        self.wave_color = wave_color or ['#ff0000', '#0000ff']
-        self.bg_color = bg_color or '#f2f3f4'
+        self.figsize = figsize or mpl.rcParams['figure.figsize']
+        self.title_font_size = title_font_size or mpl.rcParams['figure.titlesize']
+        self.wave_color = wave_color or mpl.rcParams['axes.prop_cycle'].by_key()['color'][:2]
```

Comment:
Should i get the next two colors in the cycle, or should I default to the to just the two first colors.  I probably should modify this to get the next two colors in the cycle.